### PR TITLE
tailwindcss-language-server: init at 0.0.14

### PR DESCRIPTION
**Pull request by happysalada**
## Description of changes

The fix for having the binary working with helix editor is just released, even though it doesn't appear anywhere in the release notes.

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/by-name/ta/tailwindcss-language-server/package.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  npmWorkspace = "packages/tailwindcss-language-server";
+
+  buildInputs = lib.optionals stdenv.isDarwin (with darwin.apple_sdk.frameworks; [ Security AppKit ]);
+
+  nativeBuildInputs = [ python3 ];
+
+  meta = with lib; {
+    description = "Intelligent Tailwind CSS tooling for Visual Studio Code";
+    homepage = "https://github.com/tailwindlabs/tailwindcss-intellisense";
+    license = licenses.mit; # FIXME: nix-init did not found a license
```
Comment (kirillrdy) [inline pkgs/by-name/ta/tailwindcss-language-server/package.nix:34]: according to https://www.npmjs.com/package/@tailwindcss/language-server license is MIT

**Comment by happysalada** [comment]
Result of `nixpkgs-review pr 264899` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>tailwindcss-language-server</li>
  </ul>
</details>

File: pkgs/by-name/ta/tailwindcss-language-server/package.nix (hunk 1 above)
Comment (happysalada) [reply pkgs/by-name/ta/tailwindcss-language-server/package.nix]: thanks, I just did!
I also just pushed a fix for linux-x86_64

**Review by kirillrdy — APPROVED** [review]
Result of `nixpkgs-review pr 264899` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>tailwindcss-language-server</li>
  </ul>
</details>

**Comment by happysalada** [comment]
Thank you for the review! I'll wait for ofborg before merging.

**Pull request merged**